### DISCUSSION TITLE
change mA of a driver / Cambiar los mA de un driver

### DISCUSSION THREAD

**Discussion by pauluchin** [mriscoc/Ender3V2S1#363] [Q&A] [answered]
Info
Board Creality 4.2.2 with latest firmware released

- Consult... I need to change the mA that the driver delivers to the motor... from the menus it can be done or a firmware must be compiled with this data changed... (place double Z on an ender 3 V2 with two nema motors)
Any help will be appreciated

- Consulta... necesito cambiar el mA que entrega el driver al motor.. desde los menus se puede hacer o hay que compilar un firmware con este dato cambiado... (coloque doble Z a una ender 3 V2) con dos motores nema

Cualquier ayuda sera agradecida

**Comment by mriscoc** [comment, marked as answer]
With the stock Creality boards it is not possible to control the motor driver current through the firmware. The only control is manual using the vref adjust.

**Reply by ghost** [reply]
on my ender I did the offset control on the board trimpot lowering it to 0.9v and with this procedure the motors warmed up much less than the original configuration
